infra/core/host/container-app-upsert.bicep

```diff
@@ -0,0 +1,130 @@
+metadata description = 'Creates or updates an existing Azure Container App.'
```

Comment:
FYI these will soon be pattern modules in AVM, prs are in progress, for now this is the best option.

Reply:
I saw those PRs, thanks for confirming.